src/Test/IntegrationCaseFactory.php

```diff
@@ -91,16 +91,20 @@ private function determineConfig($config)
         ]);
 
         if (!is_string($parsed['indent'])) {
-            throw new \InvalidArgumentException(sprintf(
+            throw new \InvalidArgumentException(
+                sprintf(
```

Comment:
These changes (here and above in this file) are invalid: `sprintf(` part should remain on the previous line. You may group the closing parentheses instead.